frontend/src/Editor/Components/DropDown.jsx

```diff
@@ -63,6 +63,19 @@ export const DropDown = function DropDown({
   const value = currentValueProperty ? currentValueProperty.value : '';
   const [currentValue, setCurrentValue] = useState('');
 
+  const [optionValues, setOptionValue] = useState(() => selectOptions);
+
+  useEffect(() => {
```

Comment:
We have to reset the value to default value. Here instead it seems like we're resetting to an arbitrary value: `''`. 

If the default value is not present in the latest set of options, then it might be a good idea to reset to `''`.

Reply:
yeah if the default value is not present and we should reset it to `' '`